src/utils/constants/countries.ts

```diff
@@ -0,0 +1,264 @@
+export interface CountryType {
+  code: string;
+  label: string;
+  phone: string;
+}
```

Comment:
I suggest moving this to a separate file countries.d.ts in `/src/features/common/types` and updating imports as needed